lib/blacklight.rb

```diff
@@ -39,7 +39,7 @@ def self.repository_class
     when /::/
       connection_config[:adapter].constantize
     else
-      Blacklight.const_get("#{connection_config[:adapter]}/Repository".classify)
```

Comment:
If you don't do this, you get the very confusing message that class "Repository" could not be found. I think a `KeyError` does a better job pointing to the problem